core/src/test/java/org/elasticsearch/index/IndexWithShadowReplicasIT.java

```diff
@@ -86,6 +87,7 @@
 /**
  * Tests for indices that use shadow replicas and a shared filesystem
  */
+@LuceneTestCase.AwaitsFix(bugUrl = "fix this fails intermittently")
```

Comment:
yeah - this is the fix: https://github.com/elastic/elasticsearch/pull/22631/ which is already backported to 5.x